bin/validate_submission.py

```diff
@@ -35,9 +35,10 @@ def main():
                           help='Set the script to consider all validation tasks performed as valid in the final '
                                'evaluation. This does not affect the actual report but only change the final '
                                'evaluation')
+    argparse.add_argument('--merge_per_analysis', action='store_true', default=False,
```

Comment:
Why do we want the default to be false?

Reply:
I wasn't sure whether the process was in place to merge by default, i.e. do we need to more clearly state this in our submission guidelines, etc.

@tcezard pointed out that this also means prepare backlog won't merge by default, but we could [modify the code](https://github.com/EBIvariation/eva-submission/blob/master/bin/prepare_backlog_study.py#L52) to do so if appropriate.